ztag/transforms/ftp.py

```diff
@@ -31,3 +32,40 @@ def _transform_object(self, obj):
         out['timestamp'] = obj['timestamp']
         zout.transformed = out
         return zout
+
+class FTPSTransform(ZGrabTransform):
+    name = "ftp/ftps"
+    port = 21
+    protocol = protocols.FTP
+    subprotocol = protocols.FTP.TLS
+
+    def __init__(self, *args, **kwargs):
+        super(FTPSTransform, self).__init__(*args, **kwargs)
+
+    def _transform_object(self, obj):
+        ftp_banner = obj
+        ftp = Transformable(obj)
+        zout = ZMapTransformOutput()
+        error = ftp['error'].resolve()
+        if error is not None:
```

Comment:
My reading of
http://stackoverflow.com/questions/7816363/if-a-vs-if-a-is-not-none and the
linked part of PEP8 would suggest leaving it as a comparison to None. I
think it's clearer that way as well.

On Thu, Nov 5, 2015 at 3:26 PM Zakir Durumeric <notifications@github.com>
wrote:

> In ztag/transforms/ftp.py
> <https://github.com/zmap/ztag/pull/6#discussion_r43979962>:
>
> > +
> > +class FTPSTransform(ZGrabTransform):
> > +    name = "ftp/ftps"
> > +    port = 21
> > +    protocol = protocols.FTP
> > +    subprotocol = protocols.FTP.TLS
> > +
> > +    def __init__(self, *args, **kwargs):
> > +        super(FTPSTransform, self).__init__(*args, **kwargs)
> > +
> > +    def _transform_object(self, obj):
> > +        ftp_banner = obj
> > +        ftp = Transformable(obj)
> > +        zout = ZMapTransformOutput()
> > +        error = ftp['error'].resolve()
> > +        if error is not None:
>
> can be
>
> is error:
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zmap/ztag/pull/6/files#r43979962>.
>